packages/react-impression/src/components/Select/Select.js

```diff
@@ -294,7 +294,9 @@ export default class Select extends React.PureComponent {
           option.queryChange('')
         })
         if (this.state.showOption) {
+          this.selectInner.scrollTop = 0
         } else {
+          this.selectInner.scrollTop = 0
           this.setState({ queryText: selectText })
         }
```

Comment:
this.selectInner.scrollTop = 0
if (!this.state.showOption) {
  this.setState({ queryText: selectText })
}